File: csharp/ai-news-aggregator-and-summarizer/Models/OllamaModelsDto.cs
using ai_news_aggregator_and_summarizer.Services;

namespace ai_news_aggregator_and_summarizer.Models;

public class OllamaModelsDto
{
    private List<string>? localModels = SummaryService.ListLocalModels();

    public List<string>? LocalModels
    {
        get { return localModels; }
        set { localModels = value; }
    }
}

File: java/newsscraper/src/main/java/scraper/news/models/data_transfer_objects/OllamaModelsDto.java
package scraper.news.models.data_transfer_objects;

import java.util.List;

public class OllamaModelsDto 
{
    private List<String> localModels;    

    public List<String> getLocalModels()
    {
        return localModels;
    }

    public void setLocalModels(List<String> localModels)
    {
        this.localModels = localModels;
    }
}
